benchmarks/native-beam/native-beam-spark/src/main/java/cz/seznam/euphoria/benchmarks/beam/StatelessJavaSerializer.java

```diff
@@ -64,8 +63,8 @@ public void write(Kryo kryo, Output output, Object object) {
 
   @SuppressWarnings("unchecked")
   public Object read (Kryo kryo, Input input, Class type) {
-    try {
-      return new ObjectInputStreamWithClassLoader(input, kryo.getClassLoader()).readObject();
+    try (ObjectInputStream objectStream = new ObjectInputStreamWithClassLoader(input, kryo.getClassLoader())) {
+      return objectStream.readObject();
     } catch (Exception e) {
       throw new KryoException("Error during Java deserialization.", e);
     }
```

Comment:
i'm not sure these two changes are correct. newly, the change will cause the `output/input` to be closed. and i believe this will prevent writing/reading two or more values to/from the output/input.